README.md

```diff
@@ -167,6 +167,17 @@ In the current released model, the only token-level control units are `[laugh]`,
 ## Special Appreciation
 - [wlu-audio lab](https://audio.westlake.edu.cn/) for early algorithm experiments.
 
+## Related Resources
```

Comment:
I think we can just point to https://github.com/libukai/Awesome-ChatTTS since it is updated frequently.